raster_loader/io/common.py

```diff
@@ -93,13 +95,6 @@ def rasterio_metadata(
     metadata = {}
     width = raster_info["Profile"]["Width"]
     height = raster_info["Profile"]["Height"]
-    resolution = raster_info["GEO"]["MaxZoom"]
-    metadata["block_resolution"] = resolution
-    metadata["minresolution"] = resolution
-    metadata["maxresolution"] = resolution
-    metadata["nodata"] = raster_info["Profile"]["Nodata"]
```

Comment:
That actually breaks the tests, since it changes the order of the dict. Instead of having to redo all tests, I just moved that line as well to preserve the order.